harness/determined/searcher/search_runner.py

```diff
@@ -144,18 +146,23 @@ def run_experiment(
                     else:
                         try:
                             operations = self._get_operations(event)
-                        except _ExperimentInactiveException:
+                        except _ExperimentInactiveException as ex:
                             experiment_is_active = False
+                            if ex.exp_state == determinedexperimentv1State.STATE_PAUSED:
+                                experiment_is_active = not self.pause_searcher(session)
```

Comment:
For LocalSearchRunner pausing searcher does not stop experiment but prints the warning.
For CoreSearchRunner pausing searcher, actually pause searcher experiment.